proco/schools/api.py

```diff
@@ -44,3 +45,19 @@ def get_serializer_class(self):
     @method_decorator(cache_page(timeout=settings.CACHES['default']['TIMEOUT']))
     def list(self, request, *args, **kwargs):
         return super().list(request, *args, **kwargs)
+
+
+class RandomSchoolsListAPIView(ListAPIView):
+    queryset = School.objects.prefetch_related(
```

Comment:
we don't need to color points on global map, so please remove prefetch and return only coordinates. no even need to show the names